sanity-suite/src/testBook/TestBook.js

```diff
@@ -76,17 +76,25 @@ class TestBook {
                           <pre>${JSON.stringify(testCase.inputData, undefined, 2)}</pre>
```

Comment:
Except for linting, there is no other changes in this file. Can you revert these changes?